client/README.md

```diff
@@ -0,0 +1,21 @@
+### Client
+
+Wrappers for the client used for making RPC calls with useful funcionalities like nonce tracking, multichain clients, ... 
+
+With it we can expose the contract call that we might use often using bindings from the bindings folder and objects from the crypto folder and do things like getting the hermes id from the registry smart contract, create transactions that execute smart contract calls, etc.
+
+For example in transactor we use it to register identities:
+```
+client.RegisterIdentity(td.ChainID, client.RegistrationRequest{
+    WriteRequest:    wr,
+    ChainID:         chainID,
+    Nonce:           new(big.Int).SetUint64(nonce),
+    HermesID:        common.HexToAddress(hermesAddress),
+    Stake:           stake,
+    TransactorFee:   fee,
+    Beneficiary:     common.HexToAddress(beneficiary),
+    Signature:       r.GetSignature(),
+    RegistryAddress: common.HexToAddress(registryAddress),
+})
+```
+It is worth mentioning that the client creates the transactions but doesn't send them to the blockchain. The sending is managed by other packages that also manage nonce and gas price.
```

Comment:
Oh but it can do that not problem, you don't have to use the `depot`. No one is stopping you to creating a custom nonce tracker or using the one provided and just sending transactions like that.